fixtures/favoris.ts

```diff
@@ -0,0 +1,78 @@
+import { TypeOffre, TypeRecherche } from 'interfaces/favoris'
+import { TypeOffreJson, TypeRechercheJson } from 'interfaces/json/favoris'
+
+export function uneListeDOffres() {
```

Comment:
on peut typer la fixture. On a tendance à mettre en Partial<Type>  et se donner la possibilité d'écraser la fixture par défaut pour manipuler à la volée dans nos tests.
```suggestion
export function uneListeDOffres = (overrides: Partial<Offre> = {)}: Offre => {
``

Reply:
Pour les listes on rend rarement les fixtures customisables.

Reply:
mais on peut typer!